### ISSUE TITLE
No delete of add-ons

### ISSUE THREAD

**Issue by jevers76** [dolson334/banterCustomer#4]
Add-ons can be added but not deleted

**Comment by dolson334** [comment]
@jevers76 you are correct, currently, you cannot delete. But by unselected the first check box, the add on will not show when booking. If you need an addon deleted we can delete for you. 

The reason for this, is that some of our customers are connected to QuickBooks and we want to be careful when deleting QB products.

**Comment by jevers76** [comment]
I agree that if deleting an add-on messes with Quickbooks causes problems then do not allow deletes.  I also know that  Reservation Master has a similar restriction to deleting items and over time has become has become cluttered with so many items it is difficult to find the right item.  (every item was selected in a long dropdown list)  Your use of checkboxes to hide old items is good.
